index.bs

```diff
@@ -1965,26 +1972,39 @@ Methods</h4>
 
             1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=fully active=] then return [=a promise rejected with=] "{{InvalidStateError}}" {{DOMException}}.
 
-            1. Let <var>promise</var> be a new Promise.
+            2. Let <var>promise</var> be a new Promise.
 
-            2. If the {{[[control thread state]]}} on the
+            3. If the {{[[control thread state]]}} on the
                 {{AudioContext}} is <code>closed</code> reject the
                 promise with {{InvalidStateError}}, abort these steps,
                 returning <var>promise</var>.
 
-            3. Set {{[[suspended by user]]}} to <code>false</code>.
+            4. Set {{[[suspended by user]]}} to <code>false</code>.
+
+            5. If the {{[[control thread state]]}} on the
+                {{AudioContext}} is <code>suspended</code> and there is an ongoing [=interruption=]:
 
-            4. If the context is not <a>allowed to start</a>, append
+                1. Queue a media element task</a> to execute the following steps:
```

Comment:
```suggestion
                1. <a href="https://html.spec.whatwg.org/multipage/media.html#queue-a-media-element-task">Queue a media element task</a> to execute the following steps:
```

Some parts of the spec use `[=Queue a media element task=]` instead. But I don't know which one is preferable. Maybe it depends on the context.